awx/main/tasks/jobs.py

```diff
@@ -1,77 +1,52 @@
-# -*- coding: utf-8 -*-
-
-# Copyright (c) 2015 Ansible, Inc.
-# All Rights Reserved.
+# jobs.py
 
 # Python
-from collections import OrderedDict, namedtuple, deque
+from asyncio.log import logger
+from collections import deque, OrderedDict
+from distutils.dir_util import copy_tree
 import errno
 import functools
-import importlib
+import fcntl
 import json
 import logging
 import os
-from io import StringIO
-from contextlib import redirect_stdout
+from pathlib import Path
 import shutil
 import stat
+import yaml
 import tempfile
-import time
 import traceback
-from distutils.dir_util import copy_tree
-from distutils.version import LooseVersion as Version
-import yaml
-import fcntl
-from pathlib import Path
-from uuid import uuid4
+import time
 import urllib.parse as urlparse
-import socket
-import threading
-import concurrent.futures
-from base64 import b64encode
-import sys
+from uuid import uuid4
+
 
 # Django
+from django_guid.middleware import GuidMiddleware
 from django.conf import settings
-from django.db import transaction, DatabaseError, IntegrityError
-from django.db.models.fields.related import ForeignKey
+from django.db import transaction, DatabaseError
 from django.utils.timezone import now
-from django.utils.encoding import smart_str
-from django.contrib.auth.models import User
-from django.utils.translation import ugettext_lazy as _, gettext_noop
-from django.core.cache import cache
-from django.core.exceptions import ObjectDoesNotExist
-from django_guid.middleware import GuidMiddleware
 
-# Django-CRUM
-from crum import impersonate
+
+# Runner
+import ansible_runner
 
 # GitPython
 import git
 from gitdb.exc import BadName as BadGitName
 
-# Runner
-import ansible_runner
-import ansible_runner.cleanup
```

Comment:
It's good to see groups of imports like this go away, to me, it helps justify that this makes sense.